include/appfwk/DAQModule.hpp

```diff
@@ -64,15 +64,15 @@ namespace appfwk {
  * This header also contains the definitions of the Issues that can be
  * thrown by the DAQModule.
  */
-class DAQModule : public NamedObject
```

Comment:
I feel like I would rather have NamedObjectImpl be named NamedObject, and NamedObject be named NamedObjectI, as it feels awkward to have "NamedObjectImpl" sprinkled throughout the code.